2019-10-02 10:02:08,506 ERROR 1763 [-/127.0.0.1/-/99ms POST /auth/registered] nodejs.ER_WARN_DATA_OUT_OF_RANGEError: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'phone' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:02:08', '15392129809')
code: "ER_WARN_DATA_OUT_OF_RANGE"
errno: 1264
sqlMessage: "Out of range value for column 'phone' at row 1"
sqlState: "22003"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:02:08', '15392129809')"
name: "ER_WARN_DATA_OUT_OF_RANGEError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:03:28,946 ERROR 1763 [-/127.0.0.1/-/27ms POST /auth/registered] nodejs.ER_WARN_DATA_OUT_OF_RANGEError: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'phone' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:03:28', '15392129809')
code: "ER_WARN_DATA_OUT_OF_RANGE"
errno: 1264
sqlMessage: "Out of range value for column 'phone' at row 1"
sqlState: "22003"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:03:28', '15392129809')"
name: "ER_WARN_DATA_OUT_OF_RANGEError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:07:36,439 ERROR 1763 [-/127.0.0.1/-/10ms POST /auth/registered] nodejs.ER_WARN_DATA_OUT_OF_RANGEError: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'phone' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:07:36', 15392129809)
code: "ER_WARN_DATA_OUT_OF_RANGE"
errno: 1264
sqlMessage: "Out of range value for column 'phone' at row 1"
sqlState: "22003"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:07:36', 15392129809)"
name: "ER_WARN_DATA_OUT_OF_RANGEError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:12:00,264 ERROR 1763 [-/127.0.0.1/-/13ms POST /auth/registered] nodejs.ER_WARN_DATA_OUT_OF_RANGEError: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'phone' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:12:00', 15392129809)
code: "ER_WARN_DATA_OUT_OF_RANGE"
errno: 1264
sqlMessage: "Out of range value for column 'phone' at row 1"
sqlState: "22003"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:12:00', 15392129809)"
name: "ER_WARN_DATA_OUT_OF_RANGEError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:14:06,586 ERROR 1763 [-/127.0.0.1/-/11ms POST /auth/registered] nodejs.ER_WARN_DATA_OUT_OF_RANGEError: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'phone' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:14:06', 15392129809)
code: "ER_WARN_DATA_OUT_OF_RANGE"
errno: 1264
sqlMessage: "Out of range value for column 'phone' at row 1"
sqlState: "22003"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES('jake', '111111', '2019-10-02 10:14:06', 15392129809)"
name: "ER_WARN_DATA_OUT_OF_RANGEError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:27:18,902 ERROR 1763 [-/127.0.0.1/-/4ms POST /auth/registered] nodejs.ER_BAD_NULL_ERRORError: ER_BAD_NULL_ERROR: Column 'account' cannot be null
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES(NULL, NULL, '2019-10-02 10:27:18', 15392129809)
code: "ER_BAD_NULL_ERROR"
errno: 1048
sqlMessage: "Column 'account' cannot be null"
sqlState: "23000"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES(NULL, NULL, '2019-10-02 10:27:18', 15392129809)"
name: "ER_BAD_NULL_ERRORError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:27:40,848 ERROR 1763 [-/127.0.0.1/-/8ms POST /auth/registered] nodejs.ER_BAD_NULL_ERRORError: ER_BAD_NULL_ERROR: Column 'account' cannot be null
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES(NULL, NULL, '2019-10-02 10:27:40', 15392129809)
code: "ER_BAD_NULL_ERROR"
errno: 1048
sqlMessage: "Column 'account' cannot be null"
sqlState: "23000"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `createTime`, `phone`) VALUES(NULL, NULL, '2019-10-02 10:27:40', 15392129809)"
name: "ER_BAD_NULL_ERRORError"
pid: 1763
hostname: jakedeMacBook-Air.local

2019-10-02 10:40:39,021 ERROR 2120 [-/127.0.0.1/-/44ms POST /auth/registered] nodejs.TypeError: this.ctx.service.sql.db(...).get is not a function
    at AuthService.checkPhone (/Users/jake/egg-game/init/app/service/auth.js:5:44)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:30)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:20:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:13:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 2120
hostname: jakedeMacBook-Air.local

2019-10-02 10:41:14,340 ERROR 2120 [-/127.0.0.1/-/19248ms POST /auth/registered] nodejs.TypeError: this.ctx.service.sql.db(...).get is not a function
    at AuthService.checkPhone (/Users/jake/egg-game/init/app/service/auth.js:5:44)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:30)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:20:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:13:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 2120
hostname: jakedeMacBook-Air.local

2019-10-02 10:55:55,452 ERROR 2233 [-/127.0.0.1/-/12571ms POST /auth/registered] nodejs.TypeError: Cannot read property 'length' of null
    at AuthService.checkPhone (/Users/jake/egg-game/init/app/service/auth.js:7:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2233
hostname: jakedeMacBook-Air.local

2019-10-02 10:59:09,946 ERROR 2261 [-/127.0.0.1/-/8ms POST /auth/registered] nodejs.TypeError: Cannot read property 'phone' of null
    at AuthService.checkPhone (/Users/jake/egg-game/init/app/service/auth.js:7:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2261
hostname: jakedeMacBook-Air.local

2019-10-02 10:59:37,137 ERROR 2261 [-/127.0.0.1/-/8079ms POST /auth/registered] nodejs.TypeError: Cannot read property 'phone' of null
    at AuthService.checkPhone (/Users/jake/egg-game/init/app/service/auth.js:7:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2261
hostname: jakedeMacBook-Air.local

2019-10-02 15:32:18,648 ERROR 1029 [-/127.0.0.1/-/46ms POST /auth/registered] nodejs.TypeError: Cannot read property 'ctx' of undefined
    at getCodeMsg (/Users/jake/egg-game/init/app/service/codeMsg.js:5:27)
    at userInterceptor (/Users/jake/egg-game/init/app/middleware/jwt.js:29:24)
    at fn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/middleware.js:112:12)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at /Users/jake/egg-game/init/node_modules/_egg-development@2.4.3@egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at i18n (/Users/jake/egg-game/init/node_modules/_egg-i18n@2.1.1@egg-i18n/app/middleware/i18n.js:13:12)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at dta (/Users/jake/egg-game/init/node_modules/_egg-security@2.6.1@egg-security/lib/middlewares/dta.js:12:12)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 1029
hostname: jakedeMacBook-Air.local

2019-10-02 15:33:15,279 ERROR 1029 [-/127.0.0.1/-/35670ms POST /auth/registered] nodejs.TypeError: Cannot read property 'ctx' of undefined
    at getCodeMsg (/Users/jake/egg-game/init/app/service/codeMsg.js:5:27)
    at userInterceptor (/Users/jake/egg-game/init/app/middleware/jwt.js:29:24)
    at fn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/middleware.js:112:12)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at /Users/jake/egg-game/init/node_modules/_egg-development@2.4.3@egg-development/app/middleware/egg_loader_trace.js:9:56
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at i18n (/Users/jake/egg-game/init/node_modules/_egg-i18n@2.1.1@egg-i18n/app/middleware/i18n.js:13:12)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@4.1.0@koa-compose/index.js:42:32)
    at dta (/Users/jake/egg-game/init/node_modules/_egg-security@2.6.1@egg-security/lib/middlewares/dta.js:12:12)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 1029
hostname: jakedeMacBook-Air.local

2019-10-02 15:33:36,587 ERROR 1043 [-/127.0.0.1/-/37ms POST /auth/registered] nodejs.TypeError: Cannot read property 'ctx' of undefined
    at getCodeMsg (/Users/jake/egg-game/init/app/service/codeMsg.js:5:27)
    at userInterceptor (/Users/jake/egg-game/init/app/middleware/jwt.js:29:24)
    at process._tickCallback (internal/process/next_tick.js:68:7)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 1043
hostname: jakedeMacBook-Air.local

2019-10-02 15:34:18,785 ERROR 1052 [-/127.0.0.1/-/40ms POST /auth/registered] nodejs.ER_NO_SUCH_TABLEError: ER_NO_SUCH_TABLE: Table 'db.code_err_msg' doesn't exist
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: SELECT * FROM `code_err_msg` WHERE `code` = 10002 LIMIT 0, 1
code: "ER_NO_SUCH_TABLE"
errno: 1146
sqlMessage: "Table 'db.code_err_msg' doesn't exist"
sqlState: "42S02"
index: 0
sql: "SELECT * FROM `code_err_msg` WHERE `code` = 10002 LIMIT 0, 1"
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
name: "ER_NO_SUCH_TABLEError"
pid: 1052
hostname: jakedeMacBook-Air.local

2019-10-02 16:06:38,110 ERROR 1328 [-/127.0.0.1/-/69ms POST /auth/registered] nodejs.WARN_DATA_TRUNCATEDError: WARN_DATA_TRUNCATED: Data truncated for column 'user_id' at row 1
    at Query.Sequence._packetToError (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Query.js:77:18)
    at Protocol._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    --------------------
    at Pool.query (/Users/jake/egg-game/init/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:199:23)
    at /Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as query] (/Users/jake/egg-game/init/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at RDSClient.proto._query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/jake/egg-game/init/node_modules/_ali-rds@3.3.0@ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:65:19)
    at /Users/jake/egg-game/init/node_modules/_co@4.6.0@co/index.js:54:5
    sql: INSERT INTO `account_info`(`account`, `password`, `create_time`, `phone`, `user_id`) VALUES('wangjiahui', 'xiaobudian', '2019-10-02 16:06:38', 18557530997, '8f1d4b60-e4eb-11e9-8200-a1b929f979f6')
code: "WARN_DATA_TRUNCATED"
errno: 1265
sqlMessage: "Data truncated for column 'user_id' at row 1"
sqlState: "01000"
index: 0
sql: "INSERT INTO `account_info`(`account`, `password`, `create_time`, `phone`, `user_id`) VALUES('wangjiahui', 'xiaobudian', '2019-10-02 16:06:38', 18557530997, '8f1d4b60-e4eb-11e9-8200-a1b929f979f6')"
name: "WARN_DATA_TRUNCATEDError"
pid: 1328
hostname: jakedeMacBook-Air.local

2019-10-02 17:33:06,637 ERROR 2104 [-/127.0.0.1/-/61ms POST /auth/getToken] nodejs.TypeError: #<Promise> is not iterable
    at new Set (<anonymous>)
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:7:18)
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:12:29)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2104
hostname: jakedeMacBook-Air.local

2019-10-02 17:33:06,640 ERROR 2104 [-/127.0.0.1/-/63ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: #<Promise> is not iterable' }
2019-10-02 17:33:55,312 ERROR 2104 [-/127.0.0.1/-/7594ms POST /auth/getToken] nodejs.TypeError: #<Promise> is not iterable
    at new Set (<anonymous>)
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:7:18)
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:12:29)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2104
hostname: jakedeMacBook-Air.local

2019-10-02 17:33:55,313 ERROR 2104 [-/127.0.0.1/-/7595ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: #<Promise> is not iterable' }
2019-10-02 17:34:41,950 ERROR 2104 [-/127.0.0.1/-/24ms POST /auth/getToken] nodejs.TypeError: #<Promise> is not iterable
    at new Set (<anonymous>)
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:7:18)
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:12:29)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2104
hostname: jakedeMacBook-Air.local

2019-10-02 17:34:41,951 ERROR 2104 [-/127.0.0.1/-/25ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: #<Promise> is not iterable' }
2019-10-02 17:35:01,100 ERROR 2127 [-/127.0.0.1/-/60ms POST /auth/getToken] nodejs.TypeError: #<Promise> is not iterable
    at new Set (<anonymous>)
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:8:18)
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:13:29)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2127
hostname: jakedeMacBook-Air.local

2019-10-02 17:35:01,102 ERROR 2127 [-/127.0.0.1/-/62ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: #<Promise> is not iterable' }
2019-10-02 17:49:05,661 ERROR 2257 [-/127.0.0.1/-/6510ms POST /auth/getToken] nodejs.TypeError: tokens.add is not a function
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:13:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2257
hostname: jakedeMacBook-Air.local

2019-10-02 17:49:05,665 ERROR 2257 [-/127.0.0.1/-/6514ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: tokens.add is not a function' }
2019-10-02 18:00:52,807 ERROR 2340 [-/127.0.0.1/-/50ms POST /auth/getToken] nodejs.TypeError: tokens.get is not a function
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:8:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2340
hostname: jakedeMacBook-Air.local

2019-10-02 18:00:52,810 ERROR 2340 [-/127.0.0.1/-/52ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: tokens.get is not a function' }
2019-10-02 18:01:23,195 ERROR 2344 [-/127.0.0.1/-/35ms POST /auth/getToken] nodejs.TypeError: redis.get(...).get is not a function
    at TokenService.tokens (/Users/jake/egg-game/init/app/service/token.js:7:46)
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:14:29)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:19:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2344
hostname: jakedeMacBook-Air.local

2019-10-02 18:01:23,198 ERROR 2344 [-/127.0.0.1/-/37ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: redis.get(...).get is not a function' }
2019-10-02 18:45:35,306 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 2495163ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 19:21:02,884 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 1707512ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 19:42:01,681 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 1218779ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 19:45:06,745 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 65022ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 20:04:01,012 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 430751ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 20:17:23,568 ERROR 752 nodejs.ClusterClientNoResponseError: client no response in 802544ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 752
hostname: jakedeMacBook-Air.local

2019-10-02 20:18:13,981 ERROR 2682 [-/127.0.0.1/-/4141ms POST /auth/getToken] nodejs.ReferenceError: app is not defined
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:15:5)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2682
hostname: jakedeMacBook-Air.local

2019-10-02 20:18:13,984 ERROR 2682 [-/127.0.0.1/-/4144ms POST /auth/getToken] { success: false,
  data: null,
  error: 'ReferenceError: app is not defined' }
2019-10-02 20:18:32,053 ERROR 2685 [-/127.0.0.1/-/2971ms POST /auth/getToken] nodejs.TypeError: redis.get(...).set is not a function
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:15:34)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2685
hostname: jakedeMacBook-Air.local

2019-10-02 20:18:32,056 ERROR 2685 [-/127.0.0.1/-/2973ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: redis.get(...).set is not a function' }
2019-10-02 20:40:25,801 ERROR 2874 [-/127.0.0.1/-/65ms POST /auth/getToken] nodejs.TypeError: redis.set is not a function
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:15:24)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2874
hostname: jakedeMacBook-Air.local

2019-10-02 20:40:25,804 ERROR 2874 [-/127.0.0.1/-/68ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: redis.set is not a function' }
2019-10-02 20:40:47,307 ERROR 2874 [-/127.0.0.1/-/9ms POST /auth/getToken] nodejs.TypeError: redis.set is not a function
    at TokenService.set (/Users/jake/egg-game/init/app/service/token.js:15:24)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2874
hostname: jakedeMacBook-Air.local

2019-10-02 20:40:47,308 ERROR 2874 [-/127.0.0.1/-/10ms POST /auth/getToken] { success: false,
  data: null,
  error: 'TypeError: redis.set is not a function' }
2019-10-02 22:14:40,383 ERROR 3718 [-/127.0.0.1/-/43ms POST /auth/registered] nodejs.TypeError: Cannot read property 'request' of undefined
    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:30:5)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 3718
hostname: jakedeMacBook-Air.local

2019-10-02 22:15:30,405 ERROR 3724 [-/127.0.0.1/-/34ms POST /auth/registered] nodejs.TypeError: Cannot read property 'request' of undefined
    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 3724
hostname: jakedeMacBook-Air.local

2019-10-02 22:15:30,411 ERROR 3724 [-/127.0.0.1/-/38ms POST /auth/registered] { success: false,
  data: null,
  error: 'TypeError: Cannot read property \'request\' of undefined' }
2019-10-02 22:16:48,869 ERROR 3738 [-/127.0.0.1/-/23ms POST /auth/registered] nodejs.TypeError: Cannot read property 'request' of undefined
    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 3738
hostname: jakedeMacBook-Air.local

2019-10-02 22:16:48,872 ERROR 3738 [-/127.0.0.1/-/26ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: Cannot read property \'request\' of undefined\n    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)' }
2019-10-02 22:19:06,213 ERROR 3738 [-/127.0.0.1/-/14997ms POST /auth/registered] nodejs.TypeError: Cannot read property 'request' of undefined
    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 3738
hostname: jakedeMacBook-Air.local

2019-10-02 22:19:06,214 ERROR 3738 [-/127.0.0.1/-/14999ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: Cannot read property \'request\' of undefined\n    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:11:25)\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)' }
2019-10-02 22:20:59,730 ERROR 3766 [-/127.0.0.1/-/39ms POST /auth/registered] nodejs.TypeError: Cannot read property 'app' of undefined
    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:12:25)
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)

pid: 3766
hostname: jakedeMacBook-Air.local

2019-10-02 22:20:59,735 ERROR 3766 [-/127.0.0.1/-/42ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: Cannot read property \'app\' of undefined\n    at validate (/Users/jake/egg-game/init/node_modules/_egg-validate@2.0.2@egg-validate/app/extend/context.js:12:25)\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:7)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)' }
2019-10-02 22:21:48,600 ERROR 3772 [-/127.0.0.1/-/31ms POST /auth/registered] nodejs.TypeError: this.app.validate is not a function
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:16)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18

pid: 3772
hostname: jakedeMacBook-Air.local

2019-10-02 22:21:48,604 ERROR 3772 [-/127.0.0.1/-/34ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: this.app.validate is not a function\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:16)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)\n    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18' }
2019-10-02 22:22:17,877 ERROR 3772 [-/127.0.0.1/-/10216ms POST /auth/registered] nodejs.TypeError: this.app.validate is not a function
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:16)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18

pid: 3772
hostname: jakedeMacBook-Air.local

2019-10-02 22:22:17,881 ERROR 3772 [-/127.0.0.1/-/10219ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: this.app.validate is not a function\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:16)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:34:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)\n    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18' }
2019-10-02 22:27:03,317 ERROR 3558 nodejs.ClusterClientNoResponseError: client no response in 69788ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 3558
hostname: jakedeMacBook-Air.local

2019-10-02 22:36:04,845 ERROR 3888 [-/127.0.0.1/-/41ms POST /auth/registered] nodejs.TypeError: this.validate is not a function
    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:27)
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:35:13)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18

pid: 3888
hostname: jakedeMacBook-Air.local

2019-10-02 22:36:04,848 ERROR 3888 [-/127.0.0.1/-/43ms POST /auth/registered] { success: false,
  data: null,
  error:
   'TypeError: this.validate is not a function\n    at helper.sendData (/Users/jake/egg-game/init/app/controller/auth.js:37:27)\n    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:35:13)\n    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:31:18)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)\n    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)\n    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)\n    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)\n    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)\n    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)\n    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18' }
2019-10-02 23:22:46,459 ERROR 3558 nodejs.ClusterClientNoResponseError: client no response in 60519ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/jake/egg-game/init/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 3558
hostname: jakedeMacBook-Air.local

2019-10-02 23:33:12,153 ERROR 4345 [-/127.0.0.1/-/4ms POST /auth/registered] nodejs.TypeError: Cannot read property 'length' of undefined
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:38:16)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:32:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 4345
hostname: jakedeMacBook-Air.local

2019-10-02 23:38:52,362 ERROR 4345 [-/127.0.0.1/-/9070ms POST /auth/registered] nodejs.TypeError: Cannot read property 'length' of undefined
    at Helper.sendData (/Users/jake/egg-game/init/app/extend/helper.js:38:16)
    at authController.registered (/Users/jake/egg-game/init/app/controller/auth.js:32:18)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/jake/egg-game/init/node_modules/_egg-core@4.17.3@egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
    at next (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:45:18)
    at /Users/jake/egg-game/init/node_modules/_@eggjs_router@2.0.0@@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/jake/egg-game/init/node_modules/_koa-compose@3.2.1@koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"http://127.0.0.1:8080","vary":"Origin"}
pid: 4345
hostname: jakedeMacBook-Air.local

2019-10-22 20:39:59,295 ERROR 2585 [-/127.0.0.1/-/1ms GET /] nodejs.ER_NOT_SUPPORTED_AUTH_MODEError: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Handshake.Sequence._packetToError (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:91:28)
    at Socket.<anonymous> (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:525:10)
    at Socket.emit (events.js:210:5)
    at addChunk (_stream_readable.js:308:12)
    --------------------
    at Protocol._enqueue (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/Connection.js:119:18)
    at Pool.getConnection (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:48:16)
    at /Users/wangjiahui/git/mahjong-egg/node_modules/_pify@2.3.0@pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/wangjiahui/git/mahjong-egg/node_modules/_pify@2.3.0@pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/wangjiahui/git/mahjong-egg/node_modules/_pify@2.3.0@pify/index.js:56:34)
    at Pool.query (/Users/wangjiahui/git/mahjong-egg/node_modules/_mysql@2.17.1@mysql/lib/Pool.js:202:8)
    at /Users/wangjiahui/git/mahjong-egg/node_modules/_pify@2.3.0@pify/index.js:29:7
    sql: select now() as currentTime;
code: "ER_NOT_SUPPORTED_AUTH_MODE"
errno: 1251
sqlMessage: "Client does not support authentication protocol requested by server; consider upgrading MySQL client"
sqlState: "08004"
fatal: true
name: "ER_NOT_SUPPORTED_AUTH_MODEError"
pid: 2585
hostname: wangjiahuideMacBook-Pro.local

